common/lib/xmodule/xmodule/graders.py

```diff
@@ -115,7 +116,7 @@ def grader_from_conf(conf):
                     del subgraderconf[key]
 
             subgrader = subgrader_class(**subgraderconf)
-            subgraders.append((subgrader, subgrader.category, weight))
+            subgraders.append((subgrader, subgrader.category, weight, passing_grade))
```

Comment:
Each time when we need to pass additional parameters into subgraders, we have to change this class. What if we will pass entire `subgraderconf` config?
@auraz what do you think?